.github/workflows/update-mds.yml

```diff
@@ -0,0 +1,45 @@
+name: Update README.md
+
+on:
+  pull_request_target:
+    branches:
+      - main
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{github.event.pull_request.head.ref}}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
+          token: ${{secrets.CONTANE_BOT_TOKEN}}
+      - name: Configure Git
+        run: |
+          git config user.name "contane-bot"
+          git config user.email "160241315+contane-bot@users.noreply.github.com"
+      - name: Install readme-generator-for-helm
+        run: npm install -g @bitnami/readme-generator-for-helm
+      - name: Update REAMDE.md
+        run: |
+          cd charts || exit 1
+          for chart in */; do
+            # Check if values.yaml has changed
+            if git diff --name-only HEAD~1 | grep -q "$chart/values.yaml"; then
+              echo "$chart: No values.yaml changes detected"
+              continue
+            fi
+            echo "$chart: values.yaml changes detected"
+            echo "Updating README.md for $chart"
+            readme-generator --values "$chart/values.yaml" --readme "$chart/README.md"
+            # Check for changes and commit if necessary
+            if git status -s | grep "$chart"; then
+              git add "$chart" && git commit -m "chore(${chart%*/}): Update values in README.md" # Remove trailing slash
```

Comment:
Right now it doesn't make a difference either way - since we only have one chart in this repo - but we should probably just create one commit, instead of one per chart.

I'd also be fine with opening an issue and doing that later, if and when it becomes relevant - i.e., we have more than one chart.

Reply:
Yeah, lets open an issue for that. 😂